src/components/tooltip/tooltip.js

```diff
@@ -219,13 +181,13 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
 
       // Check if we should display it or not.
       // This handles hide-* and show-* along with any user defined css
-      if ( hasComputedStyleValue('display','none') ) {
+      if ( $mdUtil.hasComputedStyleValue('display','none', element[0]) ) {
```

Comment:
Invert the method signature:  `$mdUtil.hasComputedStyleValue( target, key, defaultVal )`